sdk/include/psdk/uxtheme.h

```diff
@@ -15,6 +15,23 @@ extern "C" {
 #define DTBG_COMPUTINGREGION        0x00000010
 #define DTBG_MIRRORDC               0x00000020
 #define DTT_GRAYED                  0x00000001
+#if DLL_EXPORT_VERSION >= _WIN32_WINNT_VISTA
+#define DTT_TEXTCOLOR    0x00000001
```

Comment:
- Are these defines present in the official uxtheme header?
- Please keep the same whitespace indentation of the values as the other values above;
- `DLL_EXPORT_VERSION` shouldn't be used in the "public" headers. Are you looking for `_WIN32_WINNT` ?
And please put parentheses around the preprocessor test: `#if (whatever >= something)`